fuzzing/build.gradle.kts

```diff
@@ -0,0 +1,17 @@
+plugins {
+    id("build-logic.java")
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation(project(":sigstore-java"))
+    implementation("com.code-intelligence:jazzer-api:0.16.0")
+}
+
+tasks.register<Copy>("copyToFuzzOut") {
+    dependsOn(tasks.build)
+    into(project.property("fuzzOut")).from(sourceSets.main.get().runtimeClasspath)
```

Comment:
It would probably be better to use a fixed output here.
Currently this code breaks `./gradlew tasks` execution with https://github.com/sigstore/sigstore-java/issues/412

Reply:
this output needs to be to a specific location for fuzzing, but I added a default so this doesn't break stuff